src/helpers/getPrettierPath.js

```diff
@@ -1,14 +1,38 @@
 // @flow
 const path = require('path');
+const { sync: execaSync } = require('execa');
 const { findCached } = require('atom-linter');
+const { memoize } = require('lodash');
```

Comment:
Can we use `lodash/fp` here? Shouldn't actually affect your code but since we're using it everywhere else, it might be confusing if someone else comes along and tries to add some additional lodash imports for something they're doing.

Reply:
Sure thing, I'll update the PR with `lodash/fp`. I did not notice this was the main flavour of the lib used in the repo.